common-tools/clas-detector/src/main/java/org/jlab/detector/decode/CLASDecoder4.java

```diff
@@ -421,6 +456,18 @@ public Event getDataEvent(){
                 }
             }
         }
+        try {
```

Comment:
@baltzell here is where it is called and why I don't add it to the standard TDC list.

Reply:
ok, thanks, sounds good, just checking:)